test/unit/lookup_key_test.rb

```diff
@@ -24,7 +24,7 @@ def test_path2match_single_domain_path
     key   = ""
     value = ""
     as_admin do
-      key   = LookupKey.create!(:key => "ntp", :path => "domain", :puppetclass => Puppetclass.first)
+      key   = LookupKey.create!(:key => "ntp", :path => "domain", :puppetclass => Puppetclass.first, :key_type => "string")
```

Comment:
Can you revert these tests back to what they were before?  I don't think all of these key_types are required any more.